python/lsst/daf/butler/script/queryCollections.py

```diff
@@ -54,12 +54,7 @@ def queryCollections(repo, glob, collection_type, chains):
         A table containing information about collections.
     """
     butler = Butler(repo)
-    expression = globToRegex(glob)
-    # Only pass expression to queryCollections if there is an expression to
-    # apply; otherwise let queryCollections use its default value.
-    kwargs: Dict[str, Any] = {}
-    if expression:
-        kwargs["expression"] = expression
+    kwargs: Dict[str, Any] = {"expression": globToRegex(glob)}
```

Comment:
since you're not conditionally setting `kwargs["expression"]` you can get rid of `kwargs` and just add the argument `expression=globToRegex(glob)` to your call to `queryCollections`.